### ISSUE TITLE
Architecture Decision Records Template

### ISSUE THREAD

**Issue by koepalex** [OpenManufacturingPlatform/templates#2]
Shall we use architectural decision records to document our decisions?

[Overview and Examples of ADRs](https://github.com/joelparkerhenderson/architecture_decision_record)

**Comment by BirgitBoss** [comment]
I am always a fan to document design alternatives and then evaluate them against a given set of qualities.
However, I am not aware of a released specification that would document the alterantives that were rejected...
So this is probably just best practice for internal work in WG but not for specification?

**Comment by BirgitBoss** [comment]
See also #23  https://github.com/OpenManufacturingPlatform/rules_of_engagement/issues/23#issuecomment-636137365 concerning requirements (and now I would include qualities/non-functional requiremetns as well)

**Comment by koepalex** [comment]
> I am always a fan to document design alternatives and then evaluate them against a given set of qualities.
> However, I am not aware of a released specification that would document the alterantives that were rejected...
> So this is probably just best practice for internal work in WG but not for specification?

@BirgitBoss  yes you are right, I would suggest to use the ADRs for internal work and not as part of the specification.

**Comment by koepalex** [comment]
> See also #23 [OpenManufacturingPlatform/rules_of_engagement#23 (comment)](https://github.com/OpenManufacturingPlatform/rules_of_engagement/issues/23#issuecomment-636137365) concerning requirements (and now I would include qualities/non-functional requiremetns as well)

Agreed we should (have to) include qualities and non-functional requirements.
